### PR TITLE
FIX: check for empty strings

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: src/test/java/mops/presentation/form/FileQueryFormTest.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+        FileQuery expected = FileQuery.builder()
+                .owner("ich")
+                .build();
+
+        FileQuery fileQuery = fileQueryForm.toQuery();
+
+        assertThat(fileQuery).isEqualTo(expected);
+    }
+}
```
Comment (iTitus) [inline src/test/java/mops/presentation/form/FileQueryFormTest.java]: missing new line

**Comment by Segelzwerg** [comment]
> fix pmd warnings

too late :wink:

**Comment by KaFe23** [comment]
> > fix pmd warnings
> 
> too late 😉

F

File: src/test/java/mops/presentation/form/FileQueryFormTest.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package mops.presentation.form;
+
+import mops.businesslogic.file.query.FileQuery;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FileQueryFormTest {
```
Comment (iTitus) [inline src/test/java/mops/presentation/form/FileQueryFormTest.java]:
```suggestion
class FileQueryFormTest {
```

File: src/test/java/mops/presentation/form/FileQueryFormTest.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    @Test
+    public void emptyStringTest() {
```
Comment (iTitus) [inline src/test/java/mops/presentation/form/FileQueryFormTest.java]:
```suggestion
    void emptyStringTest() {
```

**Pull request merged**